modules/govuk_jenkins/templates/jobs/build_fpm_package.yaml.erb

```diff
@@ -0,0 +1,26 @@
+---
+- scm:
+    name: build_fpm_package
```

Comment:
I think that's just the way that we've done stuff in the past but I think we should probably refactor the whole way we do SCM for Jenkins jobs at some point.